Fixed problems when BC math is missing

tangix opened the pull request:
Changed this manually in my 1.1 installation, but thought it was time to contribute the fix.
Not running BC math on my php 7.2 installation.

mstenta commented:
Merged! Thanks @tangix !

mstenta commented:
@tangix We've had a bug report that seems to be related to this change: https://github.com/farmOS/farmOS/issues/192

Have you come across that warning?  Is that the warning that was supposed to be fixed by this change? Or something new?

What version of PHP are you running?

tangix commented:
The pull request addressed the issues I saw on my installation with my configuration of areas. I think I remember seeing other issues with some areas that I fixed (hacked) in my own install of 1.1. When 1.2 was released I created the pull request.

I'm running 7.2.18 without bc math on Debian 9.9. When I tried installing bcmath I completely messed up the php installation on the system and haven't retried this since then.

mstenta commented:
Thanks @tangix - the other issue went away, so I think it's all good!
